ui/src/components/search-result.tsx

```diff
@@ -1,149 +1,177 @@
+/* 
+This file contains the code for the <SearchResult/> Component.
+See "SearchResultProps" on this document for the the properties this component
+uses.
+*/
 
 import React from 'react';
 
+// If not working, run "npm install react-icons"
```

Comment:
You can remove that. react-icons is in the package.json so when doing `npm install` it should be just fine